docs/src/main/sphinx/installation/cli.rst

```diff
@@ -381,7 +381,14 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
-.. _cli-output-format:
+Auto suggestion
+---------------
+As you type commands in the CLI , you see offered a suggestion after the cursor
+based on history of commands already entered.
+
+Press the :kbd:→ key to accept the suggestion and replace the current command 
+line buffer. Continue typing to reject the suggestion.
```

Comment:
Could you also mention "option + →" (accept a keyword suggestion)?

Reply:
@ebyhr "option+→" accepts only the first keyword and does not accept the entire suggestion. Do you still want me to add it?

Reply:
I think completion per word is sometimes useful when we want to change the following query.

Reply:
Got it. I will add it